lms/startup.py

```diff
@@ -60,6 +62,10 @@ def run():
         # register InstructorService (for deleting student attempts and user staff access roles)
         set_runtime_service('instructor', InstructorService())
 
+    # Configure our base URL for CDN-served static assets if enabled.
+    if settings.FEATURES.get('ENABLE_STATIC_ASSET_CDN'):
+        StaticContent.base_url = settings.STATIC_ASSET_CDN_BASE_URL
```

Comment:
Fair point.  We can just have the base URL value default to an empty string and it'll work the same.  I'll make the change.